### PR TITLE
Adds legal statements to the copyright

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2788](https://openedx.atlassian.net/browse/DOC-2788)

Adds the CC-by-SA attribution to the copyright statement in all edX guides.

### Reviewers
- [x] Subject matter expert: Tena Herlihey
- [ ] Subject matter expert: @jbarciauskas 
- [x] Doc team review (sanity check): @catong @srpearce @pdesjardins 

FYI @shnayder @dan-f @dsjen 

### Testing
- [x] Ran make html without warnings or errors

### Other Version
- [ ] Google doc with screen shots of proposed changes: https://docs.google.com/a/edx.org/document/d/1FFV3LaufM-sn6aRYZeVdb90m5nQyUakEYT2qxMHgZ-A/edit?usp=sharing

### Post-review
- [ ] Add description to release notes task as a comment
- [x] Squash commits

File: docs/en_us/dashboard/source/conf.py [hunk 1]
```diff
@@ -30,7 +30,8 @@
 
 # General information about the project.
 project = u'Using edX Insights'
-copyright = u'2016, edX'
+
+copyright = u'2016, edX Inc. All rights reserved. This work is licensed under a Creative Commons Attribution 4.0 International License'
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/conf.py]: Add a period at the end of the string?

Comment (pdesjardins) [inline docs/en_us/dashboard/source/conf.py]: Also, maybe "a Creative Commons Attribution 4.0 International License" should be "the Creative Commons ...".

Unless there are more than one Creative Commons Attribution 4.0 International Licenses, I think our content is licensed under the one license.

**Comment by pdesjardins** [comment]
This looks good. I see that Creative Commons does have several Attribution licenses. :+1:

**Pull request merged**